### PR TITLE
add: stackblitz in online code editor category

### LINKED ISSUE #385: [Add] <StackBlitz>
### URL

https://stackblitz.com/

### Description of the above provided site

StackBlitz is an online code editor powered by WebContainers that allows users to create, collaborate, and deploy web applications in their web browser. 

### Category

Frontend

### Sub Category

online-code-editors

### PR DESCRIPTION
**Pull request by ParasSolanki**
## Fixes Issue

Closes #385 

## Changes proposed

Added StackBlitz in online code editor category

## Screenshots

image

### CODE REVIEW

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -55,6 +55,13 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   },
+  {
+    "name": "StackBlitz",
+    "description": "StackBlitz is an online code editor powered by WebContainers that allows users to create, collaborate, and deploy web applications in their web browser.",
```
Comment (rupali-codes) [inline database/frontend/online-code-editors.json:60]:
```suggestion
    "description": "StackBlitz is an online code editor powered by WebContainers that allows users to create, collaborate, and deploy web applications..",
```

**Comment by ParasSolanki** [comment]
We can use this description?

image

**Review by rupali-codes — APPROVED** [review]
lgtm, thanks

**Comment by ParasSolanki** [comment]
sorry not that the one above this

File: database/frontend/online-code-editors.json [hunk 2]
```diff
@@ -55,6 +55,13 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   },
+  {
+    "name": "StackBlitz",
+    "description": "StackBlitz is an online code editor powered by WebContainers that allows users to create, collaborate, and deploy web applications.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:60]:
```suggestion
    "description": "StackBlitz is an online code editor powered by WebContainers that allow users to create, collaborate, and deploy web applications.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ParasSolanki! I made a tweak to your description. Overall, great job! 🙂

**Review by CBID2 — APPROVED** [review]
Great job @ParasSolanki! 😄 LGTM! 👍🏾

**Comment by rupali-codes** [comment]
Thank you @CBID2 & @ParasSolanki

**Pull request merged**